src/CampaignsAdmin/CampaignDetails.tsx

```diff
@@ -165,23 +200,9 @@ export function CampaignDetails() {
     </>
   );
 }
-
-//TODO: A: Move to its own file? G:
-interface DateLabelProps {
-  date?: string;
-}
-
-const DueDateLabel = ({ date }: DateLabelProps) => {
-  const today = useMemo(() => new Date(), []);
-
-  if (!date) {
-    return <Tag color="geekblue">UNKNOW</Tag>;
-  }
-
-  const formatedDueDate = new Date(date);
-  return today < formatedDueDate ? <Tag color="green">ON TIME</Tag> : <Tag color="volcano">OVERDUE</Tag>;
-};
-
 const uploadedPhotoMessage = () => {
   message.success('Photo uploaded ');
 };
+const removePhotoMessage = () => {
```

Comment:
These two methods can be named `showUploadedPhotoMessage` and `showRemovedPhotoMessage`. 

BUT since they're only used once, just call `message.success(mensaje)` wherever you need to :)